frontend/src/metabase/query_builder/components/view/NativeQueryPreviewSidebar/NativeQueryPreviewSidebar.tsx

```diff
@@ -53,11 +51,14 @@ const NativeQueryPreviewSidebar = ({
     dispatch(setUIControls({ isNativeEditorOpen: true }));
   }, [question, query, dispatch]);
 
+  const isError = (error: unknown) =>
+    typeof error === "string" ? error : undefined;
+
   return (
     <NativeQueryPreview
       title={TITLE[engineType]}
       query={query}
-      error={error}
+      error={isError(error)}
```

Comment:
Apologies for suggesting wrong name initially :pray: 

```suggestion
      error={getErrorMessage(error)}
```

Reply:
No worries.
Thank you for the input AND for the feedback.
https://github.com/metabase/metabase/pull/40430/commits/d85b26339ec5e115311d68c5d0e1bd7db1464b86